### ISSUE TITLE
Janus error: "Unknown stream"

### ISSUE THREAD

**Issue by njam** [cargomedia/cm#2257]
We get this error from cm-janus when trying to stop a stream:
```
CM_Janus_StopStreamError: Unknown stream: 4d4c5ff7-f5ad-4968-a560-eff7a67719db in CM/Janus/Service.php on line 104
```

I guess it's expected because the stream can be already gone.

@tomaszdurka @vogdb what's the best way to handle this?
Should we ignore this in cm-janus? Or should CM parse the error message and make it a warning?

**Comment by tomaszdurka** [comment]
Not sure what is convention for services like these. I guess it's fine if stream doesn't exist if we try to remove it - so ok with ignoring.

**Comment by tomaszdurka** [comment]
As discussed with @njam 
- in cm-janus we will log a warning and return non-error response on the API
- no adjustments in CM are needed

@vogdb can you open ticket in cm-janus and close this? Thanks!

**Comment by vogdb** [comment]
Will be fixed in cargomedia/cm-janus#278
